source/languages/en/riak/community/product-advisories/appconfig-forward.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Forward Incompatibility of app.config
+project: riak
+version: 1.0.0+ 
+versions: false
+document: reference
+---
+
+Info | Value
+:----|:-----
+Date issued | April 15, 2015
+Product | Riak
+Affected Riak versions | 2.0.0+, 2.1.0+
+
+## Overview
+
+A forward incompatibility of app.config between 1.x and 2.x has been identified.
```

Comment:
Actually... This probably does express when upgrading from 1.3.x to 2.0.x...

I don't know if any of our customers have tried that, though. Hopefully none will, but my note might be incorrect in its recommendation.